ci-src-requirements.txt

```diff
@@ -1,2 +1 @@
-git+http://github.com/enthought/traits.git#egg=traits
-git+http://github.com/enthought/traitsui.git#egg=traitsui
+git+http://github.com/enthought/traitsui.git@release/7.0#egg=traitsui
```

Comment:
Do we still want to pin this version on master?

Reply:
Good catch.  It doesn't hurt right now, but we should fix.